scenes/game/local_game.gd

```diff
@@ -3611,6 +3622,10 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			_append_log_full(Enums.LogType.LogType_CharacterMovement, performing_player, "moves to %s, from space %s to %s." % [buddy_name, str(previous_location), str(performing_player.arena_location)])
 		"multiply_power_bonuses":
 			performing_player.strike_stat_boosts.power_bonus_multiplier = max(effect['amount'], performing_player.strike_stat_boosts.power_bonus_multiplier)
+		"multiply_positive_power_bonuses":
+			var power_bonuses = performing_player.strike_stat_boosts.power
+			if power_bonuses > 0:
```

Comment:
I don't think this is quite sufficient; for instance, if you boost fierce but Arakune uses his -1 power boost, strike_stat_boosts.power will be equal to the sum (1) and you'd get a total boost of +2 instead of the +3 you'd want (4 - 1). I was thinking this might be slightly annoying to implement; in some form, you'd want to keep track of positive power bonuses separately from cumulative power (ctrl+f for wherever strike_stat_boosts.power is changed and adjust your new variable if the effect's boost is positive; make sure to also adjust the part of the code that undoes boost effects)

Reply:
Also as a note, this might be the kind of thing that'd be useful to include in unit tests when you get to them

Reply:
hmmm with my coding skills i don't think i will be able to accomplish this implementation without a lot of help.

Reply:
It hopefully shouldn't be too hard; mostly just add the variable for tracking positive_power_bonuses, and whenever the normal power boost variable is changed, check if the change is positive and change positive_power_bonuses if so

Reply:
i'll give it a shot

Reply:
Please add a function that is something like add_player_power_bonus(player, amount).
Then make everything that changes the strike stat boosts call that, and record both the current power variable but also add your new variable to track positive bonuses

Reply:
you lost me at functions. If anyone of you have time can you implement this for me?